### PR TITLE
Bluetooth: controller: ull/lll: Do not feature exc more than once

### PR DESCRIPTION
**Pull request by alexsven**
Updated the controller implementation to not feature
exchange if already done once either by local or remote peer
device in an active connection session.

Relates to #11472

Signed-off-by: Alexander Svensen <alsv@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/ull_conn.c [hunk 1]
```diff
@@ -2169,6 +2169,44 @@ static inline void event_fex_prep(struct ll_conn *conn)
...
+		rx->hdr.handle = conn->lll.handle;
+		rx->hdr.type = NODE_RX_TYPE_DC_PDU;
+
+		/* prepare feature rsp structure */
+		pdu = (void *)rx->pdu;
+		pdu->ll_id = PDU_DATA_LLID_CTRL;
+		pdu->len = offsetof(struct pdu_data_llctrl, feature_rsp) +
+			   sizeof(struct pdu_data_llctrl_feature_rsp);
+		pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_FEATURE_RSP;
+		(void)memset(pdu->llctrl.feature_rsp.features[0], 0x00,
```
Comment (mtpr-ot) [inline subsys/bluetooth/controller/ll_sw/ull_conn.c]: `pdu->llctrl.feature_rsp.features[0]` is not a pointer. Use `pdu->llctrl.feature_rsp.features` or `&pdu->llctrl.feature_rsp.features[0]`

**Review by mtpr-ot — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_conn.c (hunk 1 above)
Comment (alexsven) [reply subsys/bluetooth/controller/ll_sw/ull_conn.c]: Yes, typo. Fixed and resubmitted

**Review by cvinayak — APPROVED** [review]
LGTM

**Review by carlescufi — APPROVED** [review]
I am OK with this, but can we start prefixing with the keyword `split`?:
`Bluetooth: controller: split: <description of change>`
since that is the name we use in Kconfig.

@aescolar @cvinayak @alexsven @mtpr-ot

**Comment by aescolar** [comment]
> but can we start prefixing 

@carlescufi Any name is good for me :)

**Review by mtpr-ot — APPROVED** [review]
OK

**Pull request merged**